tests/integration_test.rs

```diff
@@ -22,7 +22,7 @@ pub fn create_test_cap_with_fn_names(fn_names: Vec<&str>) -> Capability {
 fn start_holochain_instance() -> (Holochain, Arc<Mutex<TestLogger>>) {
     // Setup the holochain instance
     let wasm =
-        create_wasm_from_file("wasm-test/target/wasm32-unknown-unknown/debug/test_globals.wasm");
+        create_wasm_from_file("wasm-test/target/wasm32-unknown-unknown/release/test_globals.wasm");
```

Comment:
tests are made in debug by default, no?

Reply:
Only if you want to test what the right timeout for the initialization process is when that entails passing around 9MB instead of 500k of DNA entries between several HashMaps .. ;)

(1 minute was not always enough)

Reply:
We should never build wasm for debug.
Right now we don't have any means of taking advantage of debug info in our wasm build when running in our ribosome.

Reply:
Ok if we can't override panic! and assert! and "take advantage of debug info"